src/node_config.cc

```diff
@@ -85,6 +86,21 @@ static void Initialize(Local<Object> target,
   READONLY_TRUE_PROPERTY(target, "hasDtrace");
 #endif
 
+#if defined NODE_DEFAULT_PACKAGE_TYPE
+  READONLY_PROPERTY(target,
+                    "defaultPackageType",
+                    String::NewFromOneByte(isolate,
+                      reinterpret_cast<const unsigned char *>(
+                        NODE_DEFAULT_PACKAGE_TYPE))
+                      .ToLocalChecked());
```

Comment:
```suggestion
                    FIXED_ONE_BYTE_STRING(isolate, NODE_DEFAULT_PACKAGE_TYPE));
```

(Aside: statement continuations would be indented with 4 space, and the `*` in pointer types is left-leaning)